src/prelude/prim.mo

```diff
@@ -39,47 +39,61 @@ module Types = {
 };
 
 func abs(x : Int) : Nat { (prim "abs" : Int -> Nat) x };
-func shiftLeft(x : Nat, shift : Nat32) : Nat { (prim "lsh_Nat" : (Nat, Nat32) -> Nat) (x, shift) };
-func shiftRight(x : Nat, shift : Nat32) : Nat { (prim "rsh_Nat" : (Nat, Nat32) -> Nat) (x, shift) };
+func shiftLeft(x : Nat, shift : Nat32) : Nat {
+  (prim "lsh_Nat" : (Nat, Nat32) -> Nat)(x, shift);
+};
+func shiftRight(x : Nat, shift : Nat32) : Nat {
+  (prim "rsh_Nat" : (Nat, Nat32) -> Nat)(x, shift);
+};
 
 // for testing
 func idlHash(x : Text) : Nat32 { (prim "idlHash" : Text -> Nat32) x };
 
 // Printing
 
 func debugPrint(x : Text) { (prim "print" : Text -> ()) x };
-func debugPrintNat(x : Nat) { debugPrint (@text_of_Nat x) };
-func debugPrintInt(x : Int) { debugPrint (@text_of_Int x) };
-func debugPrintChar(x : Char) { debugPrint (charToText x) };
+func debugPrintNat(x : Nat) { debugPrint(@ text_of_Nat x) };
```

Comment:
"@ text_of_Nat" -> "@text_of_Nat"
also in line 56

Reply:
Yes this is a formatter error. Ryan has been informed

Reply:
Pushed a fix and updated the VS Code extension.

Reply:
Thank you!